inspirehep/modules/workflows/workflows/article.py

```diff
@@ -279,7 +278,6 @@
 
 POSTENHANCE_RECORD = [
     add_core,
-    add_note_entry,
```

Comment:
Uh, I didn't notice this before merging. Due to the known problem of migrating the workflow state, in order to deploy this we need to have empty-ish queues.